tgui/packages/tgui/interfaces/ChemDispenser.js

```diff
@@ -0,0 +1,439 @@
+
+  return (
+    <Section
+      fontSize="12px"
+      title={(
+        <Fragment>
+          Dispense
+          <Box
+            as="span"
+            ml={18}>
+            Icons:
+            <Button
+              width={2}
+              textAlign="center"
+              backgroundColor="rgba(0, 0, 0, 0)"
+              textColor={iconToggle ? "rgba(255, 255, 255, 0.5)" : "rgba(255, 255, 255, 1)"}
+              onClick={() => setIconToggle(false)}>
+              <Icon mr={1} name={"circle"} />
+            </Button>
+            <Button
+              width={2}
+              backgroundColor="rgba(0, 0, 0, 0)"
+              textColor={iconToggle ? "rgba(255, 255, 255, 1)" : "rgba(255, 255, 255, 0.5)"}
+              onClick={() => setIconToggle(true)}>
+              <Icon name={"tint"} />
+            </Button>
+          </Box>
+        </Fragment>
+      )}
+      buttons={(
+        <Box>
+          {"Dispense Amount: "}
+          <NumberInput
+            value={addAmount}
+            format={value => value + "u"}
+            width={4}
+            minValue={1}
+            maxValue={100}
+            onDrag={(e, value) => setAddAmount(value)} />
+        </Box>
+      )}>
+      <Section fitted backgroundColor="rgba(0,0,0,0)">
+        {(!maximumBeakerVolume || maximumBeakerVolume
+        === beakerTotalVolume) && (
+          <Modal
+            className="chem-dispenser__labels"
+            fontSize="20px"
+            mr={2}
+            p={3}>
+            <Box>
+              {!maximumBeakerVolume &&(
+                "No " + beakerName + " Inserted"
+              ) || currentBeakerName + " Full"}
+            </Box>
+          </Modal>
+        )}
+        {dispensableReagents.map((reagent, reagentIndex) => (
+          <Button
+            key={reagentIndex}
+            className="chem-dispenser__dispense-buttons"
+            position="relative"
+            backgroundColor={""}
+            align="left"
+            width="129.5px"
+            onMouseEnter={() => setHoverOverId(reagent.id)}
+            onMouseLeave={() => setHoverOverId("")}
+            disabled={maximumBeakerVolume === beakerTotalVolume}
+            lineHeight={1.75}
+            onClick={() => act("dispense", {
+              amount: addAmount, reagentId: reagent.id,
+            })}>
+            <Icon
+              pt={1}
+              style={{
+                "text-shadow": "0 0 3px #000",
+              }}
+              color={"rgba(" + reagent.colorR + "," + reagent.colorG + ", " + reagent.colorB + ", 1)"}
+              name={iconToggle ? stateMap[reagent.state].icon : "circle"} />
+            <Tooltip.Overflow width="105px" content={reagent.name}
+              style={{
+                "line-height": "15px",
+              }}>
+              {reagent.name}
+            </Tooltip.Overflow>
+          </Button>
+        ))}
+      </Section>
+      <Box italic pt={0.5}> {"Reagent ID: " + hoverOverId}</Box>
```

Comment:
Seems kinda weird to me to show the text of the button I'm hovering over when I can just look at the button I'm hovering over? Also pretty sure it doesn't account for tab-focus properly. Would prefer to remove than have busted for tab focus.